app/parse.py

```diff
@@ -1,15 +1,67 @@
+import csv
+import urllib.parse
+
+import requests
+from bs4 import BeautifulSoup
 from dataclasses import dataclass
 
 
+QUOTE_URL = "https://quotes.toscrape.com"
+
+
 @dataclass
 class Quote:
     text: str
     author: str
     tags: list[str]
 
 
+def parse_quote_tags(quote_soup: BeautifulSoup) -> list[str]:
+    tags = quote_soup.select_one(".tags")
+    return [
+        tag.text for tag in tags.select(".tag")
+    ]
+
+
+def parse_single_quote(quote_soup: BeautifulSoup) -> Quote:
+    return Quote(
+        text=quote_soup.select_one(".text").text,
+        author=quote_soup.select_one(".author").text,
+        tags=parse_quote_tags(quote_soup)
+    )
+
+
+def parse_quotes(quotes_soup: BeautifulSoup) -> [Quote]:
```

Comment:
Wrong return annotation

Reply:
parse_quotes function does return list of Quote or am i missing something?

Reply:
`-> list[Quote ]`